src/components/MovieListSmall.js

```diff
@@ -0,0 +1,70 @@
+import styled from 'styled-components';
+
+import useStore from '../hooks/useStore';
+
+const Section = styled.section`
+	padding: 16px;
+`;
+
+/* Used pic width's w92 w154 & w185 */
+const Article = styled.article`
+	max-width: 156px;
+	padding: 0;
+	animation: fadeIn 1s ${({delay}) => delay}s forwards;
+	border: 1px solid rgba(0, 0, 0, 0.25);
+	border-radius: 16px;
+	opacity: 0;
+	background-color: #fff;
+	box-shadow: 1px 1px 10px 5px rgba(0, 0, 0, 0.2);
+	color: #fff;
+
+	@keyframes fadeIn {
+		0% {
+			opacity: 0;
+		}
+		100% {
+			opacity: 1;
+		}
+	}
+`;
+
+const Picture = styled.img`
+	max-width: 154px;
+	border-radius: 16px 16px 0 0;
+`;
+
+const Container = styled.div`
+	display: flex;
+	flex-wrap: wrap;
+	gap: 16px;
+`;
+
+const MovieHeadline = styled.p`
+	min-height: 56px;
+	padding: 12px 8px 16px 12px;
+	color: black;
+	overflow-wrap: break-word;
+`;
+
+export default function MovieListSmall(data) {
```

Comment:
What is the difference to `MovieList`? Usually we want to avoid repeating ourselves in code. Here you repeat yourself quite a lot. Maybe think about how you could repurpose the components so that you only need one `MovieList`.

Reply:
It uses the same data but the view is limited to 4 items and the the layout is completely different.